src/deployment/deploy.py

```diff
@@ -397,8 +406,15 @@ def try_sp_create() -> None:
         ):
             set_app_audience(app.object_id, "AzureADMyOrg")
             url = "https://%s.azurewebsites.net" % self.application_name
+            api = "api://%s.azurewebsites.net" % self.application_name
+
+            if not app.identifier_uris.contains(api_url):
+                app.identifier_uris.append(api_url)
+            else:
```

Comment:
the else clause should be removed it will override the `identifier_uris` array instead of appending